### ISSUE TITLE
Lockscreen cover image doesn't go away

### ISSUE THREAD

**Issue by phpfile** [ghenry22/cordova-plugin-music-controls2#13]
On Android (8) when using this plugin with the cover parameter set to an image all seems to be working fine.

I get the cover image in the notification.
I also get the cover image on the lockscreen (when the phone is locked and I power on the screen).

Problem is when I destroy the notification. The cover image on the lockscreen does not go away.
I have to close my app for the original lockscreen image (what the user had as his default) to come back.

tldr: cover image on lockscreen is persistent even after notification destroy.

**Comment by ghenry22** [comment]
what make and model of phone are you using?  This might be something speciific to certain models as my HTC test phone doesn't actually change the lockscreen image, only the notification so I'll need to try and find a way to test this.

**Comment by phpfile** [comment]
It was a Motorola G5, I have since moved on to other projects.
Thanks for getting back to me though.
